perf/benchmark/runner/runner.py

```diff
@@ -269,10 +272,10 @@ def generate_nighthawk_cmd(self, cpus, conn, qps, duration, labels):
             "--output-format json",
             "--prefetch-connections",
             "--open-loop",
+            "--jitter-uniform 0.0001s",
```

Comment:
why does the jitter flag not impact this?

Reply:
this jitter flag is for nighthawk not for fortio.

Reply:
the introduced --jitter flag in the args is for fortio cmd.

Reply:
why does it only apply to fortio though? Why is fortio configurable but nighthawk is not?

Reply:
They both configured with jitter. 
1. nighthawk is configured with jitter through ` "--jitter-uniform 0.0001s"` 
2. fortio is configued with jitter through `--jitter=true`
two paths.

Reply:
I know, but in our own benchmark configs if we say `jitter: true` it doesn't impact nighthawk

Reply:
we can make them unified. I need to confirm the param value with Otto.